Index: UF06/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/UF06/.idea/.gitignore b/UF06/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1682421168864)
+++ b/UF06/.idea/.gitignore	(date 1682421168864)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
